### PR TITLE
UX: Changed post stream placeholder to animation

### PR DESCRIPTION
**Pull request by venarius**
This adds placeholder animation to post stream. Also enhanced the avatar animation a bit.

ezgif com-crop

Resolves: https://meta.discourse.org/t/user-more-info-cache-results-and-render-optimization/17578/27

### CODE REVIEW

**Comment by coding-horror** [comment]
Looks good, I know Kris wanted to tone down the speed and strength of the
animation a bit, can you make a first pass at that?

**Comment by venarius** [comment]
How about that?

### Before
Kapture 2019-04-25 at 9 46 59


### After
Kapture 2019-04-25 at 9 44 19

**Comment by coding-horror** [comment]
sure start with that!

File: app/assets/stylesheets/common/base/topic-post.scss [hunk 1]
```diff
@@ -1,15 +1,54 @@
+// placeholder
+@keyframes placeHolderShimmer {
```
Comment (davidtaylorhq) [inline app/assets/stylesheets/common/base/topic-post.scss]: Now that this is used in a couple of different places, can we move it somewhere more generic than `topic-post.scss`? We have a couple of other `@keyframes` in `common/base/discourse.scss`.

**Comment by davidtaylorhq** [comment]
Looks good to me, thanks @venarius

**Pull request merged**